src/azul/logging.py

```diff
@@ -163,20 +167,24 @@ def silenced_es_logger():
             assert es_log.level == original_log_level
 
 
-def http_body_log_message(body_type: str,
+type BodyType = Literal['request', 'response']
+
+
+def http_body_log_message(body_type: BodyType,
                           body: bytes | bytearray | str | IO[bytes] | IO[str] | None,
                           *,
                           verbatim: bool = False,
                           ) -> str:
     if body is None:
         return f'… without {body_type} body'
     elif isinstance(body, (bytes, bytearray, str)):
+        body_len = len(body)
         if verbatim:
             if isinstance(body, (bytes, bytearray)):
                 body = body.decode(errors='ignore')
         else:
             # https://github.com/python/typing/discussions/1911
-            body = trunc_ellipses(body, max_len=128)  # type: ignore[type-var]
-        return f'… with {body_type} body {body!r}'
+            body = trunc_ellipses(body, max_len=1024)  # type: ignore[type-var]
+        return f'… with {body_type} body {body!r} (total {body_len} bytes)'
```

Comment:
```suggestion
        prefix_len = 1024
        return f'… with first {prefix_len}/{body_len} bytes of the {body_type} body {body!r}'
```

when truncated and …